assets/stylesheets/yearly_review.scss

```diff
@@ -12,6 +12,23 @@
   width: 50%;
 }
 
+[data-review-topic-users="true"] table table tr {
+  border-top: none;
+  border-bottom: none;
+}
+
+[data-review-topic-users="true"] table table tbody {
+  border-top: none;
+}
+
+[data-review-topic-users="true"] table table td {
+  text-align: left;
+}
+
+[data-review-topic-users="true"] table table td:first-child {
+  width: 25px;
+}
```

Comment:
You can simplify this code as below since it is an SCSS.

```
[data-review-topic-users="true"] table table {
  tr {
    border-top: none;
    border-bottom: none;
  }

  tbody {
    border-top: none;
  }

  td {
    text-align: left;

    &:first-child {
      width: 25px;
    }
  }
```